auth0/testdata/recordings/TestAccClient.yaml

```diff
@@ -0,0 +1,168 @@
+---
+version: 1
+interactions:
+- request:
+    body: |
+      {"name":"Acceptance Test - TestAccClient","description":"Test Application Long Description","app_type":"non_interactive","is_first_party":true,"is_token_endpoint_ip_header_trusted":true,"oidc_conformant":true,"callbacks":["https://example.com/callback"],"allowed_origins":["https://example.com"],"web_origins":["https://example.com"],"allowed_clients":["https://allowed.example.com"],"allowed_logout_urls":["https://example.com"],"jwt_configuration":{"secret_encoded":true,"scopes":{"foo":"bar"},"alg":"RS256","lifetime_in_seconds":300},"grant_types":["authorization_code","http://auth0.com/oauth/grant-type/password-realm","implicit","password","refresh_token"],"custom_login_page_on":true,"addons":{"firebase":{"client_email":"john.doe@example.com","lifetime_in_seconds":1,"private_key":"wer","private_key_id":"qwreerwerwe"},"samlp":{"audience":"https://example.com/saml","createUpnClaim":false,"digestAlgorithm":"sha1","includeAttributeNameFormat":true,"lifetimeInSeconds":3600,"logout":{},"mapIdentities":false,"mapUnknownClaimsAsIs":false,"mappings":{"email":"http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress","name":"http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name"},"nameIdentifierFormat":"urn:oasis:names:tc:SAML:2.0:nameid-format:persistent","nameIdentifierProbes":["http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress"],"passthroughClaimsWithNoMapping":false,"signatureAlgorithm":"rsa-sha1","signingCert":"-----BEGIN PUBLIC KEY-----\nMIGf...bpP/t3\n+JGNGIRMj1hF1rnb6QIDAQAB\n-----END PUBLIC KEY-----\n","typedAttributes":true}},"token_endpoint_auth_method":"client_secret_post","client_metadata":{"foo":"zoo"},"mobile":{"ios":{"app_bundle_identifier":"com.my.bundle.id","team_id":"9JA89QQLNQ"}},"initiate_login_uri":"https://example.com/login","refresh_token":{"rotation_type":"rotating","expiration_type":"expiring","leeway":42,"token_lifetime":424242,"infinite_token_lifetime":true,"infinite_idle_token_lifetime":false,"idle_token_lifetime":3600},"organization_usage":"deny","organization_require_behavior":"no_prompt"}
```

Comment:
`signing_cert` is coming from the actual test: https://github.com/auth0/terraform-provider-auth0/blob/f88e8e05978135bb9ead62ba84dec10873439a8e/auth0/resource_auth0_client_test.go#L154